### PR TITLE
Maya

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by mayakoma7** [comment]
finish- the fylo project
remove br- html  project

File: First-Html-page/personal-site-challenge.html [hunk 1]
```diff
@@ -5,25 +5,82 @@
...
-        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci consectetur quisquam deleniti, molestiae reiciendis atque hic blanditiis suscipit voluptates harum ducimus, rerum alias! Quos cum, sequi molestiae itaque id sunt!
-        LinkedIn
-        Twitter
-        Website
+        <header class="top">
+            <div>
+                <div> <img src="asset/cloud.png" class="topCloud"></div>
+                 <h1 > I'm a Dev Club Member.</h1> 
+                 <h2> a <u>prog</u>rammer</h2>
+            <div></div><img src="asset/cloud.png" class="bottomCloud"></div>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:18]: What is this div for?

File: First-Html-page/personal-site-challenge.html [hunk 2]
```diff
@@ -5,25 +5,82 @@
...
+       
+           <span><img src="asset/mountain.png" class="mountain"></span>
+           </div>
+
+        
+        </header>
+        <div class="main">
+       <div>
+          <img src="img/94007621.png" class="profil">
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:27]: Please try to keep the indentation right

File: First-Html-page/personal-site-challenge.html [hunk 3]
```diff
@@ -5,25 +5,82 @@
...
+       <h3>Hello.</h3> 
+       <p>I am a Web Developer.And I ❤️ to code </p>
+       </div>
+
+       <span class="dot"></span>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:32]: what is this for?

File: First-Html-page/personal-site-challenge.html [hunk 4]
```diff
@@ -5,25 +5,82 @@
...
+       <span class="dot"></span>
+    
+        <h3>My Skills.</h3>
+    <div class="flex">
+    <div>
+        <div class="gif1">
+            <iframe src="https://giphy.com/embed/ZVik7pBtu9dNS" frameBorder="0"   allowFullScreen></iframe>
+        </div>
+        <div class="div1">
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:44]: Try to come up with better naming I don't know what the content of div1 is just by reading its name

File: First-Html-page/personal-site-challenge.html [hunk 5]
```diff
@@ -5,25 +5,82 @@
...
+       <span class="dot"></span>
+       <span class="dot"></span>
+       <span class="dot"></span>
+       <span class="dot"></span>
+    
+        <h3>My Skills.</h3>
+    <div class="flex">
+    <div>
+        <div class="gif1">
+            <iframe src="https://giphy.com/embed/ZVik7pBtu9dNS" frameBorder="0"   allowFullScreen></iframe>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:42]: This is not how we show an image in HTML

File: First-Html-page/personal-site-challenge.html [hunk 6]
```diff
@@ -5,25 +5,82 @@
...
+       <p> Lorem ipsum dolor sit, amet consectetur adipisicing elit. <br/>
+        Adipisci consectetur quisquam deleniti, molestiae reiciendis atque hic blanditiis suscipit voluptates harum ducimus,<br/> 
+        rerum alias! Quos cum, sequi molestiae itaque id sunt!</p>
+      
+        <button type="button">Contact Me!</button>
+       
+    </div>
+        <footer>
+           
+              <h6><span class="link">LinkedIn</span> <span class="link">Twitter</span> <span>Website</span></h6>
```
Comment (Sa-ar) [inline First-Html-page/personal-site-challenge.html:82]: nice, although I think it needs to be links

File: First-Html-page/style.css [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+    text-align: center;
+}
+.main{
+    margin-left: 300px;
+    margin-right: 300px;
+}
+.topCloud{
+    margin-bottom: 30px;
+    position: relative;
+    right: 300px;
```
Comment (Sa-ar) [inline First-Html-page/style.css:18]: is this right for every screen size?

File: First-Html-page/style.css [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+    top:40px;
+    float: right;
+
+   object-fit: cover;
+  object-position: 15% 100%;
```
Comment (Sa-ar) [inline First-Html-page/style.css:23]: you should find a better way to position this class (one that is more clear to the reader)

File: First-Html-page/style.css [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+    
+}
+.bottomCloud{
+    position: relative;
+    left:250px;
+    top:100px;
+    float: left;
```
Comment (Sa-ar) [inline First-Html-page/style.css:30]: you don't need this float. there is a better way to position things that is not float

File: First-Html-page/style.css [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+h4{
+    font-family: 'Montserrat';
+    color: #11999E;
+}
+
+h5{
+    color:#EAF6F6;
+}
+h6{
+    font-family: 'Montserrat';
```
Comment (Sa-ar) [inline First-Html-page/style.css:77]: There is a duplication in the code regarding the font family. try to keep it more DRY

File: First-Html-page/style.css [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+    height: 300px;
+    position: relative;
+    top:100px;
+    right: 70px;
+}
+
+.div1{
+   
+    text-align: left;
+    direction: ltr;
```
Comment (Sa-ar) [inline First-Html-page/style.css:91]: It's the default value, why did you need it?

File: Fylo-FronedMentor-Challenge/index.html [hunk 12]
```diff
@@ -3,20 +3,37 @@
...
   <title>Fylo data storage component</title>
 </head>
 
-<body>
-
-  You’ve used 815 GB of your storage
+<body class="main">
+<img src="assets/bg-desktop.png" class="desktopImg">
+<div class="div1">
+<img src="file:///C:/Users/mayak/OneDrive/Desktop/%D7%9C%D7%99%D7%9E%D7%95%D7%93%D7%99%D7%9D/%D7%9E%D7%95%D7%A2%D7%93%D7%95%D7%9F%20%D7%9E%D7%A4%D7%AA%D7%97%D7%99%D7%9D/week-1/Fylo-FronedMentor-Challenge/assets/logo.svg" frameBorder="0" class="fylo">
```
Comment (Sa-ar) [inline Fylo-FronedMentor-Challenge/index.html:15]: This will definitely not work on my computer...

**Review by Sa-ar — CHANGES_REQUESTED** [review]
there are a few comments. Also, you should have used flexbox for the fylo project

File: First-Html-page/personal-site-challenge.html (hunk 3 above)
Comment (mayakoma7) [reply First-Html-page/personal-site-challenge.html:32]: the span?  its for  the dots

File: First-Html-page/personal-site-challenge.html (hunk 5 above)
Comment (mayakoma7) [reply First-Html-page/personal-site-challenge.html:42]: this is the code for  the gif..  there is  another way to upload this to the web?
